src/tests.rs

```diff
@@ -385,6 +385,38 @@ macro_rules! tests {
             assert_eq!($t::<5, 10>::MAX.saturating_add(1), $t::<5, 10>::MAX);
         )*}
 
+        #[test]
+        fn wrapping_add() {$(
```

Comment:
Can you move the `$(` to the following line? This makes it clearer where the scope starts upon a quick look.

Reply:
fix [66dad44](https://github.com/jhpratt/deranged/pull/9/commits/66dad44270da41570e43294ad21f217b4e07942f)